suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -137,7 +137,14 @@ export const selectMainnetAccounts = memoize((state: AccountsRootState) =>
     ),
 );
 
-export const selectAccountByKey = (state: AccountsRootState, accountKey: AccountKey) => {
+export const selectNumberOfAccounts = (state: AccountsRootState) => selectAccounts(state).length;
+
+export const selectUserHasAccounts = (state: AccountsRootState): boolean =>
+    pipe(selectAccounts(state), A.isNotEmpty);
+
+export const selectAccountByKey = (state: AccountsRootState, accountKey?: AccountKey) => {
```

Comment:
Why did you made key optional?

Reply:
Because I'm now using this selector in a components, where the `accountKey` come from a prop as `AccountKey | undefined`. Solution would be to check that the value is present before calling the selector, but selecors are hooks and can not be called conditionally. 

We have possibility to pass undefined to other selectors, I believe this doesn't do any harm.